### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by jw3126**
Main goal of this PR is to make sure that all functionality needed in the examples is provided directly by FEniCS.jl and no PyCall is needed in the examples.

### CODE REVIEW

File: examples/tutorial9.jl [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
+"""
+FEniCS tutorial demo program: Convection-diffusion-reaction for a system
+describing the concentration of three species A, B, C undergoing a simple
+first-order reaction A + B --> C with first-order decay of C. The velocity
+is given by the flow field w from the demo navier_stokes_cylinder.py.
+  u_1' + w . nabla(u_1) - div(eps*grad(u_1)) = f_1 - K*u_1*u_2
+  u_2' + w . nabla(u_2) - div(eps*grad(u_2)) = f_2 - K*u_1*u_2
+  u_3' + w . nabla(u_3) - div(eps*grad(u_3)) = f_3 + K*u_1*u_2 - K*u_3
+"""
+module Tutorial9
```
Comment (ChrisRackauckas) [inline examples/tutorial9.jl]:
```suggestion
module Tutorial
```

**Comment by ChrisRackauckas** [comment]
Nice! I like it. Noticed one small typo. 

Also, I think it would be nice to share your example that mixes FEniCS + DiffEq's time stepping. IIRC you got that working?

File: examples/tutorial9.jl (hunk 1 above)
Comment (jw3126) [reply examples/tutorial9.jl]: The tutorials live in modules `Tutorial1` ... `Tutorial9`. So they are isolated and can be `include(...)` into the same file without warning.

**Comment by jw3126** [comment]
Yes I am planning to do that. I ran it in a situation with known analytic solution and get some deviation. I need to investigate/fix that and then will provide a PR.

File: examples/tutorial9.jl (hunk 1 above)
Comment (ChrisRackauckas) [reply examples/tutorial9.jl]: oh my bad! Missed that.

**Comment by ChrisRackauckas** [comment]
Maybe the a mass matrix needs to be set?

**Pull request merged**